code/modules/clothing/under/jobs/medsci.dm

```diff
@@ -120,8 +120,8 @@
 
 
 /obj/item/clothing/under/rank/paramedic
-	desc = "Jumpsuit of Moebius Medical paramedics. It's made with reinforced fiber to offer more protection for recovery operators."
-	name = "moebius paramedic's jumpsuit"
+	desc = "Jumpsuit of NanoTrasen medical paramedics. It's made with reinforced fiber to offer more protection for recovery operators."
+	name = "NanoTrasen paramedic's jumpsuit"
```

Comment:
This change should be in your modular file. If it is, this file needs to be reverted back to the master file.